digester.go

```diff
@@ -338,6 +338,8 @@ func (d *sqlDigester) isLit(t token) (beLit bool) {
 		beLit = true
 	} else if t.lit == "*" {
 		beLit = true
+	} else if tok == null || strings.ToLower(t.lit) == "null" {
```

Comment:
```suggestion
	} else if tok == null {
```

Both `\N` and `NULL` result in the same token.